tests/integration/defs/test_e2e.py

```diff
@@ -1665,6 +1665,14 @@ def test_openai_responses(llm_root, llm_venv):
          str(test_root / "_test_openai_responses.py")])
 
 
+def test_openai_health(llm_root, llm_venv):
+    test_root = unittest_path() / "llmapi" / "apps"
+    llm_venv.run_cmd([
+        "-m", "pytest",
+        str(test_root / "_test_openai_metrics.py -k test_health")
```

Comment:
Please update to :
str(test_root / "_test_openai_metrics.py", "-k", "test_health"